src/ui/ConnectToServer/index.tsx

```diff
@@ -114,9 +115,8 @@ class ConnectToServerContainer extends React.Component<
       const { appId, url } = this.state;
       // Use SDK default (passing undefined) on an empty string.
       const baseUrl = url || DEFAULT_BASE_URL;
-      // Clear test state to invalidate the shared App cache as a work around for
+      // TODO: Clear test state to invalidate the shared App cache as a work around for
```

Comment:
From my testing, I'm unable to reproduce this and it seems fair to disable the workaround for now.